File: RxFeature/src/main/java/com/tamsiree/rxfeature/module/scaner/OnRxScanerListener.java
package com.tamsiree.rxfeature.module.scaner;


import com.google.zxing.Result;

/**
 * @author Tamsiree
 * @date 2017/9/22
 */

public interface OnRxScanerListener {
    void onSuccess(String type, Result result);

    void onFail(String type, String message);
}

File: RxFeature/src/main/java/com/tamsiree/rxfeature/scaner/OnRxScanerListener.kt
package com.tamsiree.rxfeature.scaner

import com.google.zxing.Result

/**
 * @author Tamsiree
 * @date 2017/9/22
 */
interface OnRxScanerListener {
    fun onSuccess(type: String?, result: Result?)
    fun onFail(type: String?, message: String?)
}